coilsnake/assets/structures/eb_staff_chars.yml

```diff
@@ -0,0 +1,318 @@
+# Hint: in the 'Top 8x8 tile' field the first hex digit is the row, the second is the column in Fonts/credits.png.
```

Comment:
why not just have this as two fields then (row and column)?

Reply:
I didn't even think of that... Made that change in the latest commit.